src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexParseException.cs

```diff
@@ -6,22 +6,32 @@
 namespace System.Text.RegularExpressions
 {
     [Serializable]
-    internal sealed class RegexParseException : ArgumentException
+    public sealed class RegexParseException : ArgumentException
     {
-        private readonly RegexParseError _error; // tests access this via private reflection
-
         /// <summary>Gets the error that happened during parsing.</summary>
-        public RegexParseError Error => _error;
+        public RegexParseError Error { get; }
 
         /// <summary>Gets the offset in the supplied pattern.</summary>
         public int Offset { get; }
 
-        public RegexParseException(RegexParseError error, int offset, string message) : base(message)
+        public RegexParseException(RegexParseError error, int offset)
+        {
+            Error = error;
+            Offset = offset;
+        }
+
+        internal RegexParseException(RegexParseError error, int offset, string message) : base(message)
```

Comment:
Chain ctor above?

Reply:
Sorry, I don't understand, you mean you want the order of these ctors to be inverse of what they are now?